core/src/main/java/org/elasticsearch/action/bulk/BulkItemRequest.java

```diff
@@ -55,25 +56,16 @@ public String index() {
         return request.indices()[0];
     }
 
-    BulkItemResponse getPrimaryResponse() {
+    // NOTE: public for testing only
+    public BulkItemResponse getPrimaryResponse() {
```

Comment:
hmm. any chance to make this protected and use a test only subclass to increase visibility?